docs/docs/monitoring/analytics/getting-started-with-analytics.mdx

```diff
@@ -0,0 +1,209 @@
+---
+id: getting-started-with-analytics
+sidebar_label: Getting started
+title: Getting started with Analytics
+description:
+abstract: Rasa Pro Analytics is a data pipeline that extracts model, training,
+  and conversation data from the Rasa platform and streams the data to
+  your data warehouse for analysis.
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import RasaProLabel from "@theme/RasaProLabel";
+import RasaProBanner from "@theme/RasaProBanner";
+import {Grid} from "@theme/Grid"
+
+<RasaProLabel />
+
+<RasaProBanner />
+
+Analytics helps visualize and process Rasa assistant metrics in the
+tooling (BI tools, data warehouses) of your choice. Visualizations
+and analysis of the production assistant and its conversations allow
+you to assess ROI and improve the performance of the assistant over time.
+
+<Grid  columns="1fr 1fr" noMargin noGap verticalAlign="middle">
+  <img
+    alt="Rasa Pro Analytics - Sessions per channel"
+    src={useBaseUrl("/img/analytics/graph-number-sessions-channel.png")}
+    width="100%"
+  />
+  <img
+    alt="Rasa Pro Analytics - Escalation rate"
+    src={useBaseUrl("/img/analytics/graph-escalation-rate.png")}
+    width="100%"
+  />
+  <img
+    alt="Rasa Pro Analytics - Intents distribution over time"
+    src={useBaseUrl("/img/analytics/graph-intent-distribution.png")}
+    width="100%"
+  />
+  <img
+    alt="Rasa Pro Analytics - Top intents"
+    src={useBaseUrl("/img/analytics/graph-top-5-intents.png")}
+    width="100%"
+  />
+</Grid>
+
+Measuring a Rasa assistant's success and making strategic investments
+will lead to better business outcomes. It will also help you understand
+the needs of your end users and better serve them.
+
+<div align="center">
+  <img
+    alt="An overview of the components of Rasa Pro."
+    src={useBaseUrl("/img/rasa_pro_overview.png")}
+    width="100%"
+  />
+</div>
+
+Rasa Pro will connect to your production assistant using Kafka.
+The pipeline will compute analytics as soon as the docker container is successfully
+deployed and connected to your data warehouse and Kafka instances.
+
+
+## Prerequisites
+
+- A production deployment of Kafka is required to set up Rasa Pro.
+  We recommend using [Amazon Managed Streaming for
+  Apache Kafka](https://aws.amazon.com/msk/).
+- A production deployment of a data lake needs to be connected to
+  the data pipeline. Rasa Pro supports the following data lakes:
+
+  - [PostgreSQL](https://aws.amazon.com/rds/postgresql/) (**recommended**. All PostgreSQL >= 11.0 are supported)
+  - [Amazon Redshift](https://aws.amazon.com/redshift/)
+
+  We recommend managed deployments of your data lake to minimize maintenance
+  efforts.
+
+## Connect an assistant
+
+The `<DATA_LAKE_URL>` should have a DBAPI format, e.g.
```

Comment:
This is out of the blue. The `<DATA_LAKE_URL>` where exactly?

Reply:
nah this paragraph needs to be written this is copy paste